packages/shared/src/utils/date/date-time.ts

```diff
@@ -50,6 +50,15 @@ export const toMoment = (value?: moment.MomentInput): moment.Moment => {
 };
 
 export const toLocalFormat = (time: moment.MomentInput) => moment.utc(time).local().format('YYYY-MM-DD HH:mm:ss Z');
+export const getLongDate = (time: number): string => {
+    if (time === undefined || time === null) {
```

Comment:
just a question: why do we need this check if time can be only of type `number`?

Reply:
@maryia-deriv , no need if we use this function only with TS.
I'll remove it from here